src/mon/CommandHandler.cc

```diff
@@ -41,3 +43,40 @@ int CommandHandler::parse_bool(std::string_view str, bool* result, std::ostream&
     return -EINVAL;
   }
 }
+
+int CommandHandler::parse_hex(std::string hex_string, std::string &bin_string, unsigned int max_bits, std::ostream& ss)
```

Comment:
```suggestion
int CommandHandler::hex2bin(std::string hex_string, std::string &bin_string, unsigned int max_bits, std::ostream& ss)
```

I think this can be a private method in `MDSMap`.